index.html

```diff
@@ -0,0 +1,463 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"
+    integrity="sha384-Bfad6CLCknfcloXFOyFnlgtENryhrpZCe29RTifKEixXQZ38WheV+i/6YWSzkz3V" crossorigin="anonymous" />
+    <link rel="icon" href="./favicon.ico" />
+    <script src="./js/index.js"></script>
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Tekle Gebreyohannes</h1>
+      <button onclick="openMenu()" type="button" id="menu-bars" class="menu-bars">
```

Comment:
- [ ] Please use an event listener in your index.js file to execute this function, instead of onclick="openMenu", it's a best practice to separate your javascript code and styles from HTML.